### ISSUE TITLE
[Quest][Terokkar Forest] Vessels of Power

### ISSUE THREAD

**Issue by Talthos** [Atlantiss/NetherwingBugtracker#7042]
[//]: # (Enclose links to things related to the bug using http://wowhead.com or any other TBC database.)
[//]: # (You can use screenshot ingame to visual the issue.)
[//]: # (Write your tickets according to the format:)
[//]: # ([Quest][Azuremyst Isle] Red Snapper - Very Tasty!)
[//]: # ([NPC] Magistrix Erona)
[//]: # ([Spell][Mage] Fireball)
[//]: # ([Npc][Drop] Ghostclaw Lynx)
[//]: # ([Web] Armory doesnt work)
[//]: # (To find server revision type `.server info` command in the game chat.)

**Description**: 
Inconsistent behaviour for the clickable quest objects.

**Current behaviour**:
The clickable objects for this quest currently behave in one of 3 random ways when clicked:

- AoE explosion that hits nearby players and NPCs alike.

- Literally nothing happens.

- Yields a quest item.

**Expected behaviour**: 
The result of clicking one of the objects is supposed to be predetermined, upon the object spawning.

[Wowhead comment regarding how to know which of the clickable objects are 'trapped', and which ones always yield the quest item.](https://www.wowhead.com/quest=10028/vessels-of-power#comments:id=45912)

`You can easily detect if a vessel is a booby trap by hovering your mouse over it - real vessels change to a drastically lighter shade than the trapped ones. Since the real/fake state is predetermined, however, you may end up in a situation when you have to loot the booby traps just to get the real ones to spawn.`

In short: 'Trapped' objects don't get highlighted if you mouse over them, while the 'not-trapped' ones that yield the quest items _do_ get fully highlighted.

But as noted on Wowhead, this can occasionally result in situations where the trapped objects need to be clicked, in hopes that a not-trapped one will respawn in its place.

In addition, there are zero comments suggesting that the objects should ever yield _literally nothing,_ whether trapped or otherwise.

**Server Revision**: 
3385

Below are the actual, in-game clickable quest objects in question (presumably, the one with no loot table is the 'trapped' one):
[Non-trapped](https://www.wowhead.com/object=182581/draenei-vessel)

[Trapped](https://www.wowhead.com/object=182583/draenei-vessel)

**Comment by Wolffenstein** [comment]
The trapped version doesn't drop anything. As far as I remember chests being empty were blizzlike back in vanilla/tbc (in wotlk+ the loot window would not show up instead).

I'm closing for now as I believe to be working correctly for this expansions, but you can open a new report if you find different evidence.
